diff --git a/target/linux/ipq60xx/files-4.4/arch/arm64/boot/dts/qcom/qcom-ipq6000-cpu-opp.dtsi b/target/linux/ipq60xx/files-4.4/arch/arm64/boot/dts/qcom/qcom-ipq6000-cpu-opp.dtsi
--- a/target/linux/ipq60xx/files-4.4/arch/arm64/boot/dts/qcom/qcom-ipq6000-cpu-opp.dtsi	2022-04-04 04:45:37.000000000 +0800
+++ b/target/linux/ipq60xx/files-4.4/arch/arm64/boot/dts/qcom/qcom-ipq6000-cpu-opp.dtsi	2022-08-31 20:42:42.000000000 +0800
@@ -20,6 +20,9 @@
 		864000   1100000
 		1056000  1100000
 		1200000  1100000
+		1320000  1100000
+		1440000  1100000
+		1608000  1100000
 	>;
 	clock-latency = <200000>;
 };
@@ -30,6 +33,9 @@
 		864000   1100000
 		1056000  1100000
 		1200000  1100000
+		1320000  1100000
+		1440000  1100000
+		1608000  1100000
 	>;
 	clock-latency = <200000>;
 };
@@ -40,6 +46,9 @@
 		864000   1100000
 		1056000  1100000
 		1200000  1100000
+		1320000  1100000
+		1440000  1100000
+		1608000  1100000
 	>;
 	clock-latency = <200000>;
 };
@@ -50,6 +59,9 @@
 		864000   1100000
 		1056000  1100000
 		1200000  1100000
+		1320000  1100000
+		1440000  1100000
+		1608000  1100000
 	>;
 	clock-latency = <200000>;
 };
